### DISCUSSION TITLE
Guide for integrating with Astro

### DISCUSSION THREAD

**Discussion by LunaticMuch** [chakra-ui/chakra-ui#8317] [General]
### Description

It would be useful to have a quick guide for integrating with [Astro](https://astro.build)

### Problem Statement/Justification

Astro differs slightly from Next.js or Gatsby, so it would be good to have a dedicated guide

### Proposed Solution or API

-

### Additional Information

I am happy to work on a PR for this if there's an interest

**Comment by segunadebayo** [comment]
Use a React island. `client:load` the tree that wraps `ChakraProvider`. There is no Astro-specific package.

Same setup as Vite: https://chakra-ui.com/docs/get-started/frameworks/vite
